Gemfile.lock

```diff
@@ -19,28 +19,28 @@ GEM
     execjs (2.7.0)
     faraday (0.11.0)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.17)
+    ffi (1.9.18)
```

Comment:
just out of curiosity, all these changes were made by ruby/bundle whatever ruby thing?